packages/lesswrong/components/common/CloudinaryImage.tsx

```diff
@@ -25,6 +34,7 @@ const CloudinaryImage = ({width, height, publicId}: {
     crop="fill"
     gravity="custom"
     {...sizeProps}
+    {...imgProps}
```

Comment:
I played around with these values to see if I liked any of the other options better, and ended up not using them, but seems reasonable to keep the changes in this file.